custom_components/sat/sensor.py

```
@@ -42,20 +41,105 @@ async def async_setup_entry(_hass: HomeAssistant, _config_entry: ConfigEntry, _a
     if coordinator.supports_relative_modulation_management:
         _async_add_entities([SatCurrentPowerSensor(coordinator, _config_entry)])
 
         if float(_config_entry.options.get(CONF_MINIMUM_CONSUMPTION) or 0) > 0 and float(_config_entry.options.get(CONF_MAXIMUM_CONSUMPTION) or 0) > 0:
             _async_add_entities([SatCurrentConsumptionSensor(coordinator, _config_entry)])
 
 
+class SatRequestedSetpoint(SatClimateEntity, SensorEntity):
+
+    @property
+    def name(self) -> str:
+        return f"Requested Setpoint {self._config_entry.data.get(CONF_NAME)}"
```

Ruff (0.14.8)

`CONF_NAME` may be undefined, or defined from star imports

(F405)